css-text/Overview.bs

```diff
@@ -755,6 +755,9 @@ Languages and Typesetting</h4>
          the UA must either <a>hang</a> the <a>white space</a>
          or visually collapse their character advance widths
          such that they don't take up space in the line.
+         However, if 'overflow-wrap' is set to ''break-spaces'',
```

Comment:
When 'break-spaces' is specified, spaces should break. But this would not happen if they have already been collapsed, so I am trying to (normatively) make it explicit that you cannot do that. I thought this text was easier to understand here than in the definition of the break-spaces value. Do you think it should be moved? Do you have a suggestion of a better way to phrase it?